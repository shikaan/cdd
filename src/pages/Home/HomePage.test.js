import React from 'react'
import {shallow} from 'enzyme';
import 'jest-styled-components';

import HomePage from ".";

describe('HomePage', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HomePage/>)

    expect(wrapper).toMatchSnapshot()
  })
})
