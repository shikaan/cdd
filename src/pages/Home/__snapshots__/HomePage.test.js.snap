// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders correctly 1`] = `
<MainTemplate>
  <FeaturedArticle
    content={
      Object {
        "shared": Object {
          "readingTime": "min read",
        },
      }
    }
  />
  <OtherArticles
    content={
      Object {
        "shared": Object {
          "readingTime": "min read",
        },
      }
    }
    otherArticles={Array []}
  />
</MainTemplate>
`;
